src/VisualStudio/Core/Def/Implementation/TableDataSource/TableItem.cs

```diff
@@ -29,22 +30,27 @@ public TableItem(T item, Func<T, int> keyGenerator)
 
         public TableItem(IEnumerable<TableItem<T>> items)
         {
-#if DEBUG
-            // If code reached here,
-            // There must be at least 1 item in the list
-            Contract.ThrowIfFalse(items.Count() > 0);
+            var itemCount = items.Count();
```

Comment:
I think I could do if (!items.Any()) rather than count here.

Reply:
We would still need to compare the original count against the new count.